<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Production Planner</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabShape">
       <enum>QTabWidget::TabShape::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>设备管理</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="7" column="1">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0" rowspan="8">
            <widget class="QTableWidget" name="equips"/>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="btn_add_equip">
             <property name="text">
              <string>添加</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QPushButton" name="btn_save_equip">
             <property name="text">
              <string>保存</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QPushButton" name="btn_cancel_edit_equip">
             <property name="text">
              <string>取消</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QPushButton" name="btn_del_equip">
             <property name="text">
              <string>删除</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QPushButton" name="btn_edit_equip_color">
             <property name="text">
              <string>编辑颜色</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="btn_edit_equip">
             <property name="text">
              <string>编辑</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="4">
            <widget class="QPushButton" name="btn_add_tech">
             <property name="text">
              <string>添加工艺</string>
             </property>
            </widget>
           </item>
           <item row="3" column="4">
            <widget class="QPushButton" name="btn_del_tech">
             <property name="text">
              <string>删除工艺</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" rowspan="7">
            <widget class="QTableWidget" name="steps"/>
           </item>
           <item row="1" column="4">
            <widget class="QPushButton" name="btn_edit_tech">
             <property name="text">
              <string>修改工艺</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2" rowspan="7">
            <widget class="QTableWidget" name="techs"/>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="btn_add_step">
             <property name="text">
              <string>添加工序</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="btn_edit_step">
             <property name="text">
              <string>修改工序</string>
             </property>
            </widget>
           </item>
           <item row="6" column="4">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="4">
            <widget class="QPushButton" name="btn_save_tech">
             <property name="text">
              <string>保存修改</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="btn_save_step">
             <property name="text">
              <string>保存修改</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QPushButton" name="btn_del_step">
             <property name="text">
              <string>删除工序</string>
             </property>
            </widget>
           </item>
           <item row="4" column="4">
            <widget class="QPushButton" name="btn_cancel_tech">
             <property name="text">
              <string>取消</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QPushButton" name="btn_cancel_step">
             <property name="text">
              <string>取消</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>存货管理</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTableWidget" name="stocks"/>
        </item>
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QPushButton" name="btn_add_stock">
             <property name="text">
              <string>增加</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_edit_stock">
             <property name="text">
              <string>编辑</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_save_stock">
             <property name="text">
              <string>保存</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_del_stock">
             <property name="text">
              <string>删除</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_cancel_stock">
             <property name="text">
              <string>取消</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>加工计划</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QWidget" name="widget_4" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="btn_creat_plan">
             <property name="text">
              <string>新建</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_man_plan">
             <property name="text">
              <string>管理</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QComboBox" name="comb_plan_id">
             <property name="styleSheet">
              <string notr="true">QComboBox {
    border: 1px solid gray;   /* 边框 */
    border-radius: 3px;   /* 圆角 */
    padding: 1px 18px 1px 3px;   /* 字体填衬 */
    background: transparent;
}</string>
             </property>
             <property name="sizeAdjustPolicy">
              <enum>QComboBox::SizeAdjustPolicy::AdjustToContents</enum>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_date">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btn_update_plan">
             <property name="text">
              <string>更新</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_output">
             <property name="text">
              <string>导出</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="loads"/>
        </item>
        <item>
         <widget class="QTableWidget" name="plans"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
